src/sentry/api/serializers/models/userreport.py

```diff
@@ -12,34 +12,39 @@ def get_attrs(self, item_list, user):
         queryset = list(EventUser.objects.filter(
             id__in=[i.event_user_id for i in item_list],
         ))
+
         event_users = {e.id: d for e, d in zip(queryset, serialize(queryset, user))}
 
+        events_list = Event.objects.filter(
```

Comment:
This query needs to be scoped with a project_id. event_id is only unique within a single project, and not globally unique. This means you could pull events from different projects, which is entirely undesired. 

Looking at this, I think we need to do the same with the above EventUser query as well. I’d have to look into it a bit closer to make sure that’s true.

Reply:
@mattrobenolt Isn't `event_id` a randomly generated hash? Why is it unique by project but not globally?

Reply:
`event_id` is supplied by the SDK. So we can't really safely guarantee it. Our UNIQUE constraint on the database also only enforces it as `(project_id, event_id)` together. This means if I use some weird way to generate my own event_ids, I can't conflict with any other project, or do anything to affect anyone else.